base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2873,7 +2888,7 @@ impl fmt::Display for MetadataValue {
 }
 
 fn run_migrations(db: &LMDBDatabase) -> Result<(), ChainStorageError> {
-    const MIGRATION_VERSION: u64 = 2;
+    const MIGRATION_VERSION: u64 = 3;
```

Comment:
```suggestion
    const MIGRATION_VERSION: u64 = 2;
```
What have you changed to necessitate a version upgrade?